[*] Running ...
[*] Dataset loaded: E:\Projects\CO\reevo-main/problems/mkp_aco\dataset/train100_dataset.npz with 5 instances.
D:\Anaconda3\envs\reevo\Lib\site-packages\numpy\core\fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
D:\Anaconda3\envs\reevo\Lib\site-packages\numpy\core\_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
[*] Instance 0: 12.07676679065129
[*] Instance 1: 23.256757234822114
[*] Instance 2: 18.218517625596814
[*] Instance 3: 12.551295638492356
[*] Instance 4: 41.405277264279555
[*] Average:
21.50172291076843
